connector/src/test/scala/com/microsoft/kusto/spark/KustoSinkSchemaAdjustmentE2E.scala

```diff
@@ -0,0 +1,112 @@
+package com.microsoft.kusto.spark
+
+import com.microsoft.kusto.spark.KustoTestUtils.KustoConnectionOptions
+import com.microsoft.kusto.spark.datasink.KustoSinkOptions
+import org.apache.spark.sql._
+import org.apache.spark.sql.types.StringType
+import org.junit.runner.RunWith
+import org.scalatest.junit.JUnitRunner
+import org.scalatest.{BeforeAndAfterAll, BeforeAndAfterEach, FlatSpec}
+
+
+@RunWith(classOf[JUnitRunner])
+class KustoSinkSchemaAdjustmentE2E extends FlatSpec
+  with BeforeAndAfterEach with BeforeAndAfterAll {
+
+  private val nofExecutors = 4
+  private val testTablePrefix = "KustoBatchSinkE2E_SchemaAdjust"
+  private val spark = SparkSession.builder()
+    .appName("KustoSink")
+    .master(f"local[$nofExecutors]")
+    .getOrCreate()
+
+  private val expectedNumberOfRows = 3
+  private def newRow(index: Int): String = s"row-$index"
+
+  val appId: String = System.getProperty(KustoSinkOptions.KUSTO_AAD_APP_ID)
+  val appKey: String = System.getProperty(KustoSinkOptions.KUSTO_AAD_APP_SECRET)
+  val authority: String = System.getProperty(KustoSinkOptions.KUSTO_AAD_AUTHORITY_ID)
+  val cluster: String = System.getProperty(KustoSinkOptions.KUSTO_CLUSTER)
+  val database: String = System.getProperty(KustoSinkOptions.KUSTO_DATABASE)
+  val kustoConnectionOptions: KustoConnectionOptions = KustoConnectionOptions(cluster, database, appId, appKey, authority)
+
+
+  override def afterAll(): Unit = {
+    spark.sparkContext.stop()
+  }
+
+  override def afterEach(): Unit = {
+    KustoTestUtils.cleanup(kustoConnectionOptions, testTablePrefix)
+  }
+
+
+
+  "Source DataFrame schema adjustment" should "not adjust" in {
```

Comment:
tag all tests with 
taggedAs KustoE2E 
our triggered tests currently dont run the E2E